XRFoundation/XREngine
# Volumetric input check

## Summary
Adds player input checks to volumetric component functions.

## References
addresses #6321 

## QA Steps
Try entering the globe theater location and check if there are errors being thrown before or during first user input.
